src/app/components/Document/index.jsx

```diff
@@ -22,6 +22,8 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
         <meta name="robots" content="noindex,nofollow" />
         <meta name="theme-color" content={C_POSTBOX} />
         {meta}
+        <link rel="icon" href="/favicon.ico" type="image/x-icon" />
```

Comment:
Does this second `icon` link need to exist?

Reply:
Ooops. Good point, it should be covered by the line below.